webcommon/libs.jstestdriver/build.xml

```diff
@@ -25,7 +25,7 @@
 
     <target name="ext-compile" depends="build-init">
         <mkdir dir="build/extclasses"/>
-        <javac srcdir="extsrc" destdir="build/extclasses" deprecation="${build.compiler.deprecation}" debug="${build.compiler.debug}" source="1.6" target="1.8">
+        <javac srcdir="extsrc" destdir="build/extclasses" deprecation="${build.compiler.deprecation}" debug="${build.compiler.debug}" release="8">
```

Comment:
If I read the ant source correctly, this is not correct on JDK 8. You need to set `source`, `target` and `release` to get the desired behavior:

- on JDK 8, `release` will be ignored, as it is invalid for the compiler
- on JDK 9+, `source` and `target` are ignored

Reply:
in theory yes. but this would be only a problem if we would compile on jdk 7 or lower too. 8 would have src and target at 8 by default anyway. Everything else can use release already. @neilcsmith-net pointed this out which i didn't consider either initially.